### PR TITLE
hooks: delete console-conf hack

### PR DESCRIPTION
**Pull request by anonymouse64**
This was always meant to be temporary, so we should delete this as this has landed upstream in core/bionic branch for core18.

See https://github.com/CanonicalLtd/subiquity/pull/389 for the upstream PR for the bionic branch and https://github.com/CanonicalLtd/subiquity/pull/637 for the master-branch subiquity branch (which won't do anything for core18, since it builds off bionic anyways)

### CODE REVIEW

**Comment by mvo5** [comment]
Thanks for this!

**Comment by anonymouse64** [comment]
Hmm, let's not merge this until we can confirm that the upstream branch in subiquity (core/bionic) actually was released to the archive somewhere, as it appears the most recent release of subiquity in bionic (and hence console-conf) was from before the previously merged PR was merged.

**Comment by xnox** [comment]
This is now in https://launchpadlibrarian.net/489672152/subiquity_0.0.29+18.04.1_0.0.29+18.04.2.diff.gz at https://launchpad.net/~canonical-foundations/+archive/ubuntu/ubuntu-image/+packages?field.name_filter=subiquity&field.status_filter=published&field.series_filter=bionic

version 18.04.2

**Review by xnox — APPROVED** [review]

**Pull request merged**
